pages/api/arcade/shop.js

```diff
@@ -1,37 +1,45 @@
 import AirtablePlus from "airtable-plus"
 
 export const shopParts = async () => {
+  const baseID = "app4kCWulfB02bV8Q"
   const shopItemsTable = new AirtablePlus({
     apiKey: process.env.AIRTABLE_API_KEY,
-    baseID: "app4kCWulfB02bV8Q",
+    baseID,
     tableName: "Shop Items"
   })
   const ordersTable = new AirtablePlus({
     apiKey: process.env.AIRTABLE_API_KEY,
-    baseID: "app4kCWulfB02bV8Q",
+    baseID,
     tableName: "Orders"
   })
```

Comment:
We can replace this with one AirtablePlus class, and pass `tableName` to the read calls to increase perf